schemas/abstract/MPModel.json

```diff
@@ -18,15 +12,6 @@
         },
         "prototype": {
           "$ref": "strings.json#/definitions/_id"
-        },
-        "bundled": {
-          "type": "boolean"
-        },
-        "keywordIDs": {
-          "type": "array",
-          "items": {
-            "$ref": "strings.json#/definitions/keywordID"
-          }
```

Comment:
are we sure about that, as I can see we use it in `manuscripts-api` and in FE(`body-editor`, `article-editor`)

Reply:
@asouqi , can you give more information? is it just usage in the code, or is it actually being used.

Reply:
> is it just usage in the code

 API: 
  [syncAccessControl](https://github.com/Atypon-OpenSource/manuscripts-api/blob/966e9456996c6ac5525085f710e586074f1deaf3/src/DataAccess/syncAccessControl.ts#L261)
[DatabaseIndices](https://github.com/Atypon-OpenSource/manuscripts-api/blob/966e9456996c6ac5525085f710e586074f1deaf3/src/DataAccess/DatabaseIndices.ts#L73)
 
 manuscripts-editor:
[KeywordsInput]( https://github.com/Atypon-OpenSource/manuscripts-article-editor/blob/e14e67c586a2de92bb7f472e6e113be9bcee45d1/src/components/projects/KeywordsInput.tsx#L35)
 [CategorisedKeywordsInput](https://github.com/Atypon-OpenSource/manuscripts-article-editor/blob/e14e67c586a2de92bb7f472e6e113be9bcee45d1/src/components/projects/CategorisedKeywordsInput.tsx#L285)
[LibraryItems]( https://github.com/Atypon-OpenSource/manuscripts-article-editor/blob/e14e67c586a2de92bb7f472e6e113be9bcee45d1/src/components/library/LibraryItems.tsx#L166)
 
 if we remove it will not have an effect on the API and editor, but if we look at requirement validation and fix for the manuscript will not work with this change:
 [validate](https://github.com/Atypon-OpenSource/manuscripts-requirements/blob/9155c63cf6dee2bcd6d0a19aad10dc2330d8f7c8/src/validate.ts#L811)
 [fix-manuscript](https://github.com/Atypon-OpenSource/manuscripts-requirements/blob/9155c63cf6dee2bcd6d0a19aad10dc2330d8f7c8/src/fix-manuscript.ts#L242)

Reply:
Thanks for the investigation. Most of these should already be not used (I don't believe the quality checks are in use right now).